package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/tiny-sdf",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Reminder to avoid changing the version in the PR, and do it separately with `npm version` in the main branch.